libcst/_nodes/base.py

```diff
@@ -109,6 +109,12 @@ def _clone(val: object) -> object:
 
 @dataclass(frozen=True)
 class CSTNode(ABC):
+
+    # pyre-ignore[4]: Attribute `__slots__` of class `CSTNode`
+    # has type `typing.Tuple[]` but no type is specified.
+    # But we know it's str
+    __slots__ = ()
```

Comment:
I think you can remove this pyre-ignore by doing
```suggestion
    __slots__: Sequence[str] = ()
```

Reply:
I needed an additional `ClassVar` to prevent it from being picked up as a dataclass field